tools/bumbershoot/idpassemble/idpassemble.xml

```diff
@@ -0,0 +1,122 @@
+            <param name="filter_at_gene_level_condition.min_spectra" value="2" />
+            <param name="filter_at_gene_level_condition.min_additional_peptides" value="1" />
+            <param name="min_spectra_per_distinct_match" value="1" />
+            <param name="min_spectra_per_distinct_peptide" value="1" />
+            <param name="max_protein_groups_per_peptide" value="10" />
+            <output name="output" file="../../idpassemble/test-data/201208-378803-msgf.idpDB" compare="sim_size" delta_size="500000" />
+        </test>
+        <test>
+            <param name="input" value="../../idpqonvert/test-data/201208-378803-cm.idpDB" />
+            <param name="max_fdr_score" value="0.05" />
+            <param name="filter_at_gene_level_condition.min_distinct_peptides" value="2" />>
+            <param name="filter_at_gene_level_condition.min_spectra" value="2" />
+            <param name="filter_at_gene_level_condition.min_additional_peptides" value="1" />
+            <param name="min_spectra_per_distinct_match" value="1" />
+            <param name="min_spectra_per_distinct_peptide" value="1" />
+            <param name="max_protein_groups_per_peptide" value="10" />
+            <output name="output" file="../../idpassemble/test-data/201208-378803-cm.idpDB" compare="sim_size" delta_size="500000" />
+        </test>
+        <test>
+            <param name="input" value="../../idpqonvert/test-data/201208-378803-mm.idpDB,../../idpqonvert/test-data/201208-378803-msgf.idpDB,../../idpqonvert/test-data/201208-378803-cm.idpDB" />
+            <param name="max_fdr_score" value="0.05" />
+            <param name="filter_at_gene_level_condition.min_distinct_peptides" value="2" />>
+            <param name="filter_at_gene_level_condition.min_spectra" value="2" />
+            <param name="filter_at_gene_level_condition.min_additional_peptides" value="1" />
+            <param name="min_spectra_per_distinct_match" value="1" />
+            <param name="min_spectra_per_distinct_peptide" value="1" />
+            <param name="max_protein_groups_per_peptide" value="10" />
+            <output name="output" file="../../idpassemble/test-data/201208-378803.idpDB" compare="sim_size" delta_size="500000" />
+        </test>
+    </tests>
+    <help>
+<![CDATA[
+**What it does**
+
+Merges and filters one or more IDPicker 3 idpDB files into a combined idpDB file. Protein assembly (e.g. parsimony) is conducted on the combined set of proteins.
+]]>
+    </help>
+    <citations>
+        <citation type="doi">10.1021/pr900360j</citation>
+        <citation type="bibtex">@misc{toolsGalaxyP, author = {Chilton, J, Chambers MC, et al.}, title = {Galaxy Proteomics Tools}, publisher = {GitHub}, journal = {GitHub repository},
+                                      year = {2015}, url = {https://github.com/chambm/tools-galaxyp}}</citation> <!-- TODO: fix substitution of commit ", commit = {$sha1$}" -->
```

Comment:
No. I was working on a git smudge/clean script that would replace {$sha1$} with the current commit hash so that the citations part would link to the correct github commit. It was going to be really cool, but I couldn't get git to run the script and I couldn't spend time to figure out why. So I commented it out for later work.